spaces/S000208/properties/P000003.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000208
+property: P000003
+value: true
+---
+
+$X$ is a subspace of $\prod_n (\omega_n+1)$ with box topology, which is {P3}.
```

Comment:
The definition of $x$ was using $\prod_n (\omega_{n+1}+1)$ because the index was starting at $n=0$.
But this is ok if we assume $\prod_{n\ge 1}$ (which is actually easier and what Rudin uses in her paper; on the other hand S138 was already written to use $n\ge 0$, so probably easier to continue with that).
What do you think?

Reply:
Basically, $X$ is a subspace of a box product of Hausdorff spaces.